WakaTime.py

```diff
@@ -37,23 +41,15 @@
 PLUGIN_DIR = os.path.dirname(os.path.realpath(__file__))
 API_CLIENT_URL = 'https://github.com/wakatime/wakatime/archive/master.zip'
 API_CLIENT = os.path.join(PLUGIN_DIR, 'wakatime-master', 'wakatime', 'cli.py')
-SETTINGS_FILE = os.path.join(PLUGIN_DIR, 'wakatime.ini')
-SETTINGS = None
-settings = None
+SETTINGS_FILE = os.path.join(os.path.expanduser('~'), '.wakatime.cfg')
```

Comment:
Correct, all wakatime plugins share it so if you update your api key you don't have to change it for every editor.